### PR TITLE
Make SmartStore portable

### PR DESCRIPTION
**Pull request by jhsfdc**

### CODE REVIEW

**Comment by jhsfdc** [comment]
@smcnulty-sfdc @wes566 @kchitalia

File: SalesforceSDK/SmartStore/SmartStore.csproj [hunk 1]
```diff
@@ -125,11 +47,11 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>$(SolutionDir)\packages\Newtonsoft.Json.6.0.5\lib\netcore45\Newtonsoft.Json.dll</HintPath>
+      <HintPath>..\packages\Newtonsoft.Json.6.0.5\lib\portable-net45+wp80+win8+wpa81\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
```
Comment (kchitalia) [inline SalesforceSDK/SmartStore/SmartStore.csproj]: PLease use $(SolutionDir) here

File: SalesforceSDK/SmartStore/SmartStore.csproj [hunk 2]
```diff
@@ -125,11 +47,11 @@
...
     </Reference>
     <Reference Include="SQLitePCL, Version=3.8.7.2, Culture=neutral, PublicKeyToken=bddade01e9c850c5, processorArchitecture=MSIL">
-      <HintPath>$(SolutionDir)\packages\SQLitePCL.3.8.7.2\lib\netcore451\SQLitePCL.dll</HintPath>
+      <HintPath>..\packages\SQLitePCL.3.8.7.2\lib\portable-net45+sl50+win+wpa81+wp80+MonoAndroid10+xamarinios10+MonoTouch10\SQLitePCL.dll</HintPath>
```
Comment (kchitalia) [inline SalesforceSDK/SmartStore/SmartStore.csproj]: Same here.

**Comment by wes566** [comment]
:+1:

**Comment by kchitalia** [comment]
Really minor changes. Other than looks good. :+1:  :)

File: SalesforceSDK/SmartStore/SmartStore.csproj (hunk 1 above)
Comment (jhsfdc) [inline SalesforceSDK/SmartStore/SmartStore.csproj]: haha sorry

Comment (kchitalia) [inline SalesforceSDK/SmartStore/SmartStore.csproj]: I know its really easy to miss this one.

**Pull request merged**